api/v1alpha1/releaseplanadmission_types.go

```diff
@@ -59,10 +65,7 @@ type ReleasePlanAdmissionStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="Display Name",type=string,priority=1,JSONPath=`.spec.displayName`
-// +kubebuilder:printcolumn:name="Application",type=string,JSONPath=`.spec.application`
```

Comment:
Is it possible to print a list here? Having the application names show here would be quite nice

Reply:
No. This is the info printed by kubectl in a table. We would break the table easily. It will have to be a detailed view.